dcplib/networking.py

```diff
@@ -12,8 +12,9 @@
 logger = logging.getLogger(__name__)
 
 class Session(requests.Session):
+    obey_retry_after = True
     def resolve_redirects(self, resp, req, **kwargs):
-        if self.get_redirect_target(resp) and "Retry-After" in resp.headers:
+        if self.obey_retry_after and self.get_redirect_target(resp) and "Retry-After" in resp.headers:
```

Comment:
From our discussion: summarize how you're overriding urllib3 and requests to obey the Retry-After header

Reply:
Done.